files/start.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex
+
+if [ ! -f "/home/steamuser/.steam/steamapps/common/Empyrion - Dedicated Server/dedicated.yaml" ] || [ ${FORCE_APP_UPDATE} = 1 ]
+then
+	if [ ${BRANCH} == public ]
+	then
+		# GA Branch
+		/usr/games/steamcmd +login anonymous +app_update 530870 validate +quit 
+	else
+		# used specified branch
+		/usr/games/steamcmd +login anonymous +app_update 530870 -beta ${BRANCH} validate +quit 
+	fi
+fi
+
+if [ ! -f "~/Empyrion - Dedicated Server/dedicated.yaml" ]
```

Comment:
needs to be corrected, should be `~/"Empyrion - Dedicated Server"/dedicated.yaml`